dist/PublicLab.Editor.css

```diff
@@ -320,6 +320,53 @@ https://github.com/sliptree/bootstrap-tokenfield/pull/287 */
   background-color: #007bff !important;
 }
 
+@media (max-width: 800px) {
```

Comment:
Change this to about `992px` and we should be good!

Reply:
Alright then
On it right away

Reply:
But why exactly 992 if I may ask?

Reply:
Ok then, looks great on my end but lets try.
Also can you confirm it looks great and 993px and wider on your end?

Reply:
Awesome!

Reply:
Haha love this quick iteration, you two! Thanks! 🎉